stroyprombeton/tests/tests_views.py

```diff
@@ -762,14 +756,31 @@ def test_too_many_tags_collapse(self):
             )
         product.save()
 
+    # set tags limit to value "10" to check
```

Comment:
It seems the test's comment is clear, so this is excess. Decide

Reply:
@duker33 , this string is the part of the third-lined comment
```
    # set tags limit to value "10" to check
    # if tags from-to label contains max numeric, but not alphabetical value.
    # For example we should see "от 1 м до 11 м", but not "от 1 м до 9 м".
```

And this not point is not clear from the test docstring, i think.
So, i leave it, but change words order to make more clear three comment lines coupling